components/LandingPage/MusingsFromTheTeam/MusingsFromTheTeam.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from './MusingsFromTheTeam.module.scss';
+import type { BlogPost } from 'types/Base';
+import Image from '../../Image';
+
+type MusingsFromTheTeamProps = {
+    postData: BlogPost[];
+};
+
+const BlogPostAuthor = ({ post }: BlogPostCardProps) => (
+    <div className={styles.author}>
+        <Image
+            className={styles.authorPhoto}
+            src={post.author.image.url}
+            alt={post.author.image.description}
+        />
+        <div className={styles.authorDetails}>
+            <p className={styles.authorName}>{post.author.name}</p>
+            <p className={styles.authorMetadata}>
+                <span className={styles.authorRole}>{post.author.role}</span>{' '}
+                <span className={styles.authorDate}>{post.date}</span>
+            </p>
+        </div>
+    </div>
+);
+
+type BlogPostCardProps = {
+    post: BlogPost;
+};
+
+const BlogPostCard = ({ post }: BlogPostCardProps) => (
+    <li className={styles.blogPost}>
+        <a
+            className={styles.blogPostLink}
+            href={post.slug}
+            aria-label={post.title}
+        >
+            <h3 className={styles.blogPostTitle}>{post.title}</h3>
+            <BlogPostAuthor post={post} />
+        </a>
+    </li>
+);
+
+export const MusingsFromTheTeam = ({ postData }: MusingsFromTheTeamProps) => {
+    const posts = postData
+        .slice(0, 3)
```

Comment:
**Thought:** I'm not sure this slice is needed anymore if we're using the `limit` arguement on the graphql query i.e. `blogPostsCollection(limit: 3){`